src/main/java/com/patchworkmc/PatchworkUI.java

```diff
@@ -659,13 +662,14 @@ public static InputStream loadOrDownloadIntermediary(String version, File parent
 		return new FileInputStream(file);
 	}
 
+	// This class has a few unused fields at the moment, so I commented them out.
 	private static class YarnBuild {
 		String gameVersion;
-		String separator;
-		int build;
+		// String separator;
+		// int build;
```

Comment:
Have you tested this doesn’t break gson?

Reply:
gson isn't strict like jotlin serialisation, even tho I don't really want these to be commented out.

Reply:
I didn't catch that when reading for through the source, I just addressed it and added the fields back.